### PR TITLE
feat(sheets-hyper-link): add hyper link for sheet

### PR DESCRIPTION
**Pull request by weird94**
close #xxx

## Pull Request Checklist

- [x] Related tickets or issues have been linked in the PR description (or missing issue).
- [x] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [x] Unit tests have been added for the changes (if applicable).
- [x] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Review by wzhudev — CHANGES_REQUESTED** [review]
I added a custom eslint so you can catch all importing problems.

File: packages/sheets-ui/src/services/hover-manager.service.ts [hunk 1]
```diff
@@ -42,6 +42,8 @@ export class HoverManagerService extends Disposable {
         )
     )));
 
+    currentCellDebounce$ = this.currentCell$.pipe(debounceTime(100));
```
[the remark this answers is not in the document]
Comment (weird94) [reply packages/sheets-ui/src/services/hover-manager.service.ts]: fixed

**Review by wzhudev — CHANGES_REQUESTED** [review]
After some internal discussion, the implementation of the ref range service need to be updated.

**Comment by weird94** [comment]
https://github.com/dream-num/univer/pull/2330#issuecomment-2133491107

**Comment by weird94** [comment]
> ## View Deployment
> 📑 Examples	📚 Storybook
> [🔗 Preview link](https://univer-ok0phblmw-univer.vercel.app/)	[🔗 Preview link](https://univer-design-k5c65v9xd-univer.vercel.app/)

pkg

**Comment by weird94** [comment]
https://github.com/dream-num/univer/pull/2330#issuecomment-2133491107

**Review by wzhudev — APPROVED** [review]
LGTM

**Pull request merged**